test/spec/algoliasearch.helper/client.js

```diff
@@ -1,113 +1,102 @@
 'use strict';
 
-var test = require('tape');
-var sinon = require('sinon');
 var algoliaSearchHelper = require('../../../index.js');
 var version = require('../../../src/version');
 var algoliasearch = require('algoliasearch');
 
 function makeFakeClient() {
   var client = algoliasearch('what', 'wait', {});
-  client.search = sinon.stub().returns(new Promise(function() {}));
+
+  client.search = jest.fn(function() {
+    return new Promise(function() {});
+  });
+
   return client;
 }
 
-test("client without addAlgoliaAgent() doesn't throw on instanciation", function(t) {
+test("client without addAlgoliaAgent() doesn't throw on instanciation", function() {
   var client = {};
 
-  t.doesNotThrow(function() {
+  expect(function() {
     algoliaSearchHelper(client);
-  });
-
-  t.end();
+  }).not.toThrow();
 });
 
-test('addAlgoliaAgent gets called if exists', function(t) {
+test('addAlgoliaAgent gets called if exists', function() {
   var client = {
-    addAlgoliaAgent: sinon.stub()
+    addAlgoliaAgent: jest.fn()
   };
 
-  t.notOk(client.addAlgoliaAgent.called);
+  expect(client.addAlgoliaAgent).not.toHaveBeenCalled();
 
   algoliaSearchHelper(client);
 
-  t.ok(client.addAlgoliaAgent.called);
-
-  t.end();
+  expect(client.addAlgoliaAgent).toHaveBeenCalled();
```

Comment:
We should use `toHaveBeenCalledTimes(n)`.

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453